### ISSUE TITLE
`amp-video-docking` player support

### ISSUE THREAD

**Issue by alanorozco** [ampproject/amphtml#20759]
- [ ] amp-3q-player
- [ ] amp-brid-player
- [x] amp-brightcove
- [x] amp-dailymotion (great support possible, #20911)
- [x] amp-delight-player (#20913)
- [ ] ~~amp-gfycat~~ (UX N/A)
- [x] amp-ima-video
- [ ] amp-mowplayer
- [ ] amp-nexxtv-player
- [ ] amp-ooyala-player (**Issues:** no `hideControls` support)
- [ ] amp-powr-player
- [x] amp-video
- [x] amp-video-iframe
- [ ] amp-vimeo (**Issues:** no `hideControls` support)
- [ ] amp-viqeo-player
- [ ] amp-wistia-player
- [x] amp-youtube

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @alanorozco Please add an appropriate category.

**Comment by aghassemi** [comment]
we can figure this out from code. closing for housekeeping.
